app/models/concerns/searchable.rb

```diff
@@ -5,15 +5,14 @@ module Searchable
     include PgSearch
     include SearchCache
 
-    pg_search_scope :pg_search, {
-      against: :ignored, # not used since using a tsvector_column
-      using: {
-        tsearch: { tsvector_column: 'tsv', dictionary: "spanish", prefix: true }
-      },
-      ignoring: :accents,
-      ranked_by: '(:tsearch)',
-      order_within_rank: (self.column_names.include?('cached_votes_up') ? "#{self.table_name}.cached_votes_up DESC" : nil)
-    }
+    pg_search_scope :pg_search,
```

Comment:
Agree, many auto-correct indentations from rubocop doesn't look more than "hardcoded rules". But in this case what's happening is just removing braces from a hash parameter ({...}) that are not neccesary. So.. @xuanxu what should I do?

A- Revert completely the changes

B- Revert the formatting, but keep the removal of the curly braces?